hadoop-ozone/dist/src/main/compose/upgrade/compose/ha/docker-config

```diff
@@ -24,7 +24,9 @@ OZONE-SITE.XML_ozone.om.address.omservice.om1=om1
 OZONE-SITE.XML_ozone.om.address.omservice.om2=om2
 OZONE-SITE.XML_ozone.om.address.omservice.om3=om3
 OZONE-SITE.XML_ozone.om.ratis.enable=true
-
+// setting ozone.scm.ratis.enable to false for now, as scm ha upgrade is
+// not supported yet. This is supposed to work without SCM HA configuration
+OZONE-SITE.XML_ozone.scm.ratis.enable=false
```

Comment:
If upgrade to SCM HA is not yet supported, why are we enabling it by default?  Shouldn't we implement upgrade first, then enable by default?

My concern is this: consider a non SCM HA cluster, where `ozone.scm.ratis.enable=false` comes from the default config.  If upgraded, will SCM HA now be suddenly enabled after upgrade?  Will the admin need to disable SCM HA manually to make it work?

Reply:
this is safe to use, in case of non HA, it should just startup a single node ratis server. This needs to be enabled, so as to make sure the the tests excercise the HA path, not the standalone path.